Bottom: 7fd80449b6e3ff3efb57bba0e98e2196d5aa5bd8
Top:    df7551c6c1673d4c4e70cf09f45f41a8ca31a737
Author: Ian Kent <raven@themaw.net>
Date:   2013-06-06 13:48:17 +0800

Refresh of autofs-5.0.5-fix-master-map-bogus-keywork-match.patch

---

diff --git a/lib/master_parse.y b/lib/master_parse.y
index 680f116..11caf5b 100644
--- a/lib/master_parse.y
+++ b/lib/master_parse.y
@@ -75,7 +75,7 @@ static unsigned int debug;
 
 static int lineno;
 
-#define YYDEBUG 1
+#define YYDEBUG 0
 
 #ifndef YYENABLE_NLS
 #define YYENABLE_NLS 0
diff --git a/lib/master_tok.l b/lib/master_tok.l
index 064f8fd..8159245 100644
--- a/lib/master_tok.l
+++ b/lib/master_tok.l
@@ -91,7 +91,7 @@ OPTWS		[[:blank:]]*
 NL		\r?\n
 CONT		\\\n{OPTWS}
 
-OPTIONSTR	([\-]?([[:alpha:]_]([[:alnum:]_\-])*(=(["]?([[:alnum:]_\-\:])+["]?)*)+))
+OPTIONSTR	([\-]?([[:alpha:]_]([[:alnum:]_\-])*(=(["]?([[:alnum:]_\-\:])+["]?)))+)
 MACROSTR	(-D{OPTWS}([[:alpha:]_]([[:alnum:]_\-\.])*)=([[:alnum:]_\-\.])+)
 SLASHIFYSTR	(--(no-)?slashify-colons)
 NUMBER		[0-9]+
